packages/mds-geography-author-api/tests/api.spec.ts

```diff
@@ -24,14 +24,18 @@
 /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
 
 import { ApiServer } from '@mds-core/mds-api-server'
-import db from '@mds-core/mds-db'
-import { GeographyServiceManager } from '@mds-core/mds-geography-service'
 import {
-  DISTRICT_SEVEN,
+  GeographyRepository,
+  GeographyServiceClient,
+  GeographyServiceManager,
+  GeographyWithMetadataDomainModel
+} from '@mds-core/mds-geography-service'
+import {
   GEOGRAPHY2_UUID,
   GEOGRAPHY_UUID,
   LA_CITY_BOUNDARY,
   POLICY_UUID,
+  restrictedAreas,
```

Comment:
That does seem cleaner, and doesn't immediately remind me of Blomkamp's debut feature film.